# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/*/glibc/riscv32.patch.cross0
# Copyright (C) 2021 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- glibc-2.33/sysdeps/riscv/bits/wordsize.h.vanilla	2021-02-09 11:31:37.266368067 +0100
+++ glibc-2.33/sysdeps/riscv/bits/wordsize.h	2021-02-09 11:31:45.295368482 +0100
@@ -19,7 +19,7 @@
 #if __riscv_xlen == (__SIZEOF_POINTER__ * 8)
 # define __WORDSIZE __riscv_xlen
 #else
-# error unsupported ABI
+# define __WORDSIZE __riscv_xlen
 #endif
 
 #define __WORDSIZE_TIME64_COMPAT32  1
